README.md

```diff
@@ -1,28 +0,0 @@
-Stereo Vision
```

Comment:
We need to retain the main `README.md` outright.
Please edit this file instead.